### DISCUSSION TITLE
Dịch dạng động từ của "mask"

### DISCUSSION THREAD

**Discussion by trgiangdo** [d2l-ai/d2l-vi#2811] [Terminology]
Trong #2696, ngữ cảnh như sau:
> The gradients of unobserved inputs are **masked out** to ensure that only observed ratings contribute to the model learning process.

Em hiện đang dịch là
> Gradient của các đầu vào chưa biết được **lọc ra** để đảm bảo rằng ...

Anh @cuongvng và @nguyenthaibinh có đề xuất dịch là
> Gradient của các đầu vào chưa biết được **che mặt nạ** để đảm bảo rằng ...

Theo anh @cuongvng, nếu để `mặt nạ` sẽ dễ liên hệ với phần code. Em cũng đồng ý với quan điểm này.
Em chỉ sợ là người đọc khi đọc từ `che mặt nạ` sẽ hơi khó hiểu. Hiện em cũng chưa tìm được tài liệu tiếng Việt chuẩn nào có dịch từ `mask` dưới dạng động từ nên cũng khá phân vân.
Mong mọi người cho ý kiến.

cc: @duythanhvn @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair

**Comment by rootonchair** [comment]
Cá nhân từ `lọc ra` cũng đã bao hàm `masking` rồi, vì masking là một dạng filtering theo không gian. Theo mình dịch theo cách này cũng không sai. Trong bối cảnh câu trên thì sử dụng từ lọc ra giúp người đọc dễ đọc hơn thật

> Em chỉ sợ là người đọc khi đọc từ che mặt nạ sẽ hơi khó hiểu

Cái này mình không đồng ý cho lắm khi những chương trước cũng có giới thiệu khái niệm về masking. Nhất là NLP

**Reply by minhduc0711** [reply]
từ này chắc dùng nhiều thì quen thôi, ban đầu mình cũng thấy gượng :smile:

**Reply by trgiangdo** [reply]
Trong glossary của mình có nhiều từ các văn bản tiếng Việt khác cũng đâu có 😅

**Comment by lkhphuc** [comment]
> Gradient của các đầu vào chưa biết được lọc ra để đảm bảo rằng ...

`Gradient của các đầu vào chưa được quan sát sẽ được **che lại** ...`

Từ mask ngoài nghĩa face mask = mặt nạ / khẩu trang hay dùng thì còn được dịch chung `tấm chắn, tấm che`, vậy nên dịch động từ là `che khuất, che chắn, che đậy`, mình thấy từ lọc - filter không sát nghĩa bằng.

**Comment by trgiangdo** [comment]
Hiện có các đề xuất sau, mọi người vote giúp em.

1. `Lọc ra` 👍
2. `Che mặt nạ` 😄
3. `Che lại` ❤
